Routing/Loader/Reader/RestActionReader.php

```diff
@@ -333,39 +333,41 @@ public function read(RestRouteCollection $collection, \ReflectionMethod $method,
 
             // add route to collection
             $route = new Route(
-                $path, $defaults, $requirements, $options, $host, [], $methods, $condition
+                $path, $defaults, $requirements, $options, $host, [], $methods, $versionCondition
```

Comment:
Well at this point we are only adding the version condition which was defined in line 295. So I think its actually less confusing like this?